standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreListenerNotifier.java

```diff
@@ -94,6 +96,19 @@ public void notify(MetaStoreEventListener listener, ListenerEvent event) throws
               listener.onDropDatabase((DropDatabaseEvent)event);
             }
           })
+          .put(EventType.CREATE_DATACONNECTOR, new EventNotifier() {
+            @Override
+            public void notify(MetaStoreEventListener listener,
+                ListenerEvent event) throws MetaException {
+              listener.onCreateDataConnector((CreateDataConnectorEvent)event);
+            }
+          })
+          .put(EventType.DROP_DATACONNECTOR, new EventNotifier() {
+            @Override
+            public void notify(MetaStoreEventListener listener, ListenerEvent event) throws MetaException {
+              listener.onDropDataConnector((DropDataConnectorEvent)event);
+            }
+          })
```

Comment:
Do we need AlterDataConnector event or not?

Reply:
yes. I think so. Not sure why I left it out. Cannot think of any reason now. WIll add it.

Reply:
fixed.